static/skin/taskbar.js

```diff
@@ -3,6 +3,39 @@ function htmlDecode(input) {
     return doc.documentElement.textContent;
 }
 
+function handleScrollFor520Width() {
```

Comment:
Including the width value in the function name is too much detail. `handleScrollForNarrowClient()` is better but still incorrect since this function doesn't *handle* scrolling but rather *sets up* handling of scrolling. A good name is `setupAutoHidingOfTheToolbar()`. The relation to the small client isn't reflected in this proposed name since nothing in the implementation depends on the size of the window and it would do exactly the same in a large window too. The conditions under which we decide to enable this feature matter only at the call site of this function.

Reply:
Thanks, much better.